src/renderer/views/AddProject/StepOne/StepOne.tsx

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable prettier/prettier */
+import React, { useEffect, useState } from 'react';
+import ImagePicker from '../../../components/ImagePicker/ImagePicker';
+import TextField from '../../../components/TextField/TextField';
+import TextArea from '../../../components/TextArea/TextArea';
+import InputLabel from '../../../components/InputLabel/InputLabel';
+import './StepOne.scss';
+import { useTranslation } from 'react-i18next';
+
+export default function StepOne(props: any) {
+  const [imagePickerError, setImagePickerError] = useState(false);
```

Comment:
I would default this to true for now so that it is possible to move on to other components. Could add a comment above it specifying why it was done.